libs/wingsdk/src/target-sim/bucket.inflight.ts

```diff
@@ -60,7 +60,7 @@ export class Bucket implements IBucketClient, ISimulatorResourceInstance {
     return topicClient.publish(key);
   }
 
-  private async fileExists(key: string): Promise<boolean> {
+  public async exists(key: string): Promise<boolean> {
     return fs.promises
```

Comment:
please wrap it in the context function:
``` 
return this.context.withTrace({
      message: `Exists (key=${key}).`,
      activity: async () => {...}})
``` 
we may want to keep fileExists private method in order to keep one calling to `this.context.withTrace` for each public method